substrate-processor/src/processor/handlerProcessor.ts

```diff
@@ -913,6 +968,19 @@ class HandlerRunner<S> extends Runner<S, DataHandlers>{
                             log.debug('end')
                         }
                     }
+                    for (let handler of this.getEthereumTranscationHandlers(handlers, item.call)) {
+                        if (item.call.success || handler.triggerForFailedCalls) {
```

Comment:
Hey, the problem here is that successful execution of substrate call doesn't imply successful evm transaction.

https://github.com/paritytech/frontier/blob/ef75c42e9eb65319fd09ae589e3d965633b2ef51/frame/ethereum/src/lib.rs#L597

So, the processing of EVM transactions should be based on `Ethereum.Executed` event.

Reply:
Is it possible to do it on processor side or should I fix it in ingester?

Reply:
I came up with two options:
1. A mapping developer should request 'Ethereum.Executed' event itself and check if a transaction is successful
2. The substrate-gateway should annotate `call.evmTxSuccess` field like it does for `event.evmTxHash` (this option rely on network runtime which is error prone)